packages/myst-cli/src/transforms/links.ts

```diff
@@ -82,27 +82,38 @@ type LinkInfo = {
 
 export type LinkLookup = Record<string, LinkInfo>;
 
+export function getSourceFolder(link: string, sourceFile: string, projectFolder: string): string {
+  return link.startsWith('/') ? projectFolder : path.dirname(sourceFile);
+}
 /**
  * Compute link node file path relative to site root (currently, only the working directory)
  *
+ * @param pathFromLink Path from link node URL relative to file where it is defined
+ * @param file File where link is defined
+ */
+export function fileFromRelativePath(pathFromLink: string, file?: string): string | undefined {
```

Comment:
I don't think this function is used anywhere in the codebase anymore - just kept in case any external libraries are dependent on it.

Could mark as `@deprecated`?